frontend/src/metabase/collections/components/Header/Header.styled.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+import Icon from "metabase/components/Icon";
+
+export const ToggleMobileSidebarIcon = styled(Icon).attrs({
+  ml: 1,
+  mr: 2,
+  mt: "4px",
+  name: "burger",
+  size: 20,
+})`
+  color: {color("text-dark")};
+  cursor: pointer;
+
+  @media screen and (min-width: 40em) {
```

Comment:
Curious how best to have this `min-width: 40em` inside a constant as we already do, certainly outside `styled-components`.

Reply:
I guess we should use `styled-components` theming for that ideally. But for now, I think it'd be okay to use CSS variables from [here](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/css/core/breakpoints.css) maybe?